### DISCUSSION TITLE
Integrating 10x sc-multiomics (scATAC )across species

### DISCUSSION THREAD

**Discussion by JianiC** [stuart-lab/signac#1792] [Q&A]
I have 10x sc multiomics datasets (scRNA and scATAC) that from both human and mouse. I integrated scRNA modality with different references simply by gene names. I was wondering how could I integrate scATA across human and mouse. In addition, is that possible to perform WNN analysis for 10x multiome after integrating scRNA and scATAC across species

**Comment by timoast** [comment]
You would need to find equivalent ATAC-seq peaks across the two species, which is much more difficult than identifying the homologous genes
